src/assets/thirdPartyServices/snet/cntk_image_recon/index.js

```diff
@@ -125,9 +140,9 @@ class CNTKImageRecognition extends React.Component {
   renderComplete() {
     const { response } = this.state;
     var top5 = response.top_5;
-    top5 = top5.replace('{', '');
-    top5 = top5.replace('}', '');
-    var topArray = top5.split(', ');
+    top5 = top5.replace("{", "");
+    top5 = top5.replace("}", "");
+    var topArray = top5.split(", ");
```

Comment:
Suggesting to use `let` instead `var` because let will not be hoisted and block-scoped.
There is a probability that `top5` could be an empty string. It is better to handle that case in the render.

Reply:
Ok, thanks!

Reply:
Done.